pkgs/top-level/all-packages.nix

```diff
@@ -19255,7 +19257,9 @@ in
 
   sysklogd = callPackage ../os-specific/linux/sysklogd { };
 
-  syslinux = callPackage ../os-specific/linux/syslinux { };
+  syslinux = callPackage ../os-specific/linux/syslinux {
+    stdenv = gcc9Stdenv;
```

Comment:
https://bugs.gentoo.org/705730
https://gitweb.gentoo.org/repo/gentoo.git/commit/?id=8284530f4bd86a38b9867f847bd844f5d8d08e87

Reply:
It fails with ``undefined reference to `strlen'``
https://wiki.syslinux.org/wiki/index.php?title=Building